packages/opencensus-core/src/metrics/export/types.ts

```diff
@@ -0,0 +1,340 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/** Properties of a Metric which has one or more timeseries */
+export interface Metric {
+  /**
+   * The descriptor of the Metric. This is an optimization for network wire
+   * size, from data-model perspective a Metric contains always
+   * a MetricDescriptor.
+   * In case of a streaming RPC can be sent only the first time a metric is
+   * reported to save network traffic.
+   */
+  readonly metricDescriptor: MetricDescriptor;
+  /**
+   * In case of a streaming RPC this can be sent for metrics that already
+   * sent the MetricDescriptor once.
+   */
+  readonly name: string;
```

Comment:
Metric contains only MetricDescriptor and TimeSeries. Please remove ```readonly name: string;```

Reply:
It has name: [metrics.proto #L46](https://github.com/census-instrumentation/opencensus-proto/blob/b11a67434194733b34c11f206938263fa16ad1cf/src/opencensus/proto/metrics/v1/metrics.proto#L46)

Reply:
Done

Reply:
Done